lib/ts/recipe/session/types.ts

```diff
@@ -402,25 +406,27 @@ export abstract class SessionClaim<T> {
 
     /**
      * Removes the claim from the payload, by cloning and updating the entire object.
+     * If a root level prop needs to be removed from the payload this should set it to null,
+     * to have mergeIntoAccessTokenPayload remove it during the update.
      *
      * @returns The modified payload object
      */
     abstract removeFromPayload(payload: JSONObject, userContext?: any): JSONObject;
 
     /**
-     * Removes the claim from the payload, by cloning and updating the entire object.
+     * Gets the value of the claim stored in the payload
      *
-     * @returns The modified payload object
+     * @returns Claim value
      */
     abstract getValueFromPayload(payload: JSONObject, userContext: any): T | undefined;
 
-    async fetchAndSetClaim(userId: string, payload: JSONObject, userContext?: any): Promise<JSONObject> {
+    async fetchAndGetAccessTokenPayloadUpdate(userId: string, userContext?: any): Promise<JSONObject> {
```

Comment:
I'm happy to revert the name change. The only reason I updated it is that this actually returns on object intended to be used with `mergeIntoAccessTokenPayload`. This means that if the fetch returns undefined it will actually set the claim prop to null in the returned object so that it will be cleared after the merge. 
We could make this not remove claims if they are set and tell users to explicitly use `removeClaim` if they actually want to clear something from the token.